packages/iov-ethereum/src/jsonrpcclient.ts

```diff
@@ -0,0 +1,11 @@
+import { Stream } from "xstream";
+
+import { JsonRpcRequest, JsonRpcResponse } from "@iov/jsonrpc";
+import { SocketWrapperMessageEvent } from "@iov/socket";
+
+export interface JsonRpcClient {
```

Comment:
Can we call this `EthereumRpcClient`? We have JsonRpcClient already in `@iov/jsonrpc`